### PR TITLE
[docs] updatad docs of hybrid adam and cpu adam

### PR DESCRIPTION
**Pull request by Gy-Lu**

### CODE REVIEW

**Review by ver217 — APPROVED** [review]

File: colossalai/nn/optimizer/cpu_adam.py [hunk 1]
```diff
@@ -1,9 +1,51 @@
 import math
-
 import torch
 
+from colossalai.registry import OPTIMIZERS
+
 
+@OPTIMIZERS.register_module
```
Comment (feifeibear) [inline colossalai/nn/optimizer/cpu_adam.py:7]: @OPTIMIZERS.register_module

File: colossalai/nn/optimizer/cpu_adam.py [hunk 2]
```diff
@@ -1,9 +1,51 @@
...
 class CPUAdam(torch.optim.Optimizer):
+    """Implements Adam algorithm.
+
+    Supports both GPU and CPU. Requires ColossalAI to be installed via
```
Comment (feifeibear) [inline colossalai/nn/optimizer/cpu_adam.py:11]: supports (what) both resident on CPU and GPU?

File: colossalai/nn/optimizer/cpu_adam.py [hunk 3]
```diff
@@ -1,9 +1,51 @@
...
+    ``pip install .``.
+
+    This version of CPU Adam accelates parameters updating on CPU with SIMD.
+    Support of AVX2 or AVX512 is required.
+
+    The GPU part is an naive implementation.
```
Comment (feifeibear) [inline colossalai/nn/optimizer/cpu_adam.py:17]: Polish this line. part is not an implementation.

File: colossalai/nn/optimizer/cpu_adam.py [hunk 4]
```diff
@@ -1,9 +1,51 @@
...
+
+    CPU Adam also supports the hybrid calculation of fp16 and fp32.
```
Comment (feifeibear) [inline colossalai/nn/optimizer/cpu_adam.py:19]: support the (what tensor is int format of) fp16 and fp32?

File: colossalai/nn/optimizer/hybrid_adam.py [hunk 5]
```diff
@@ -1,7 +1,50 @@
...
+
+@OPTIMIZERS.register_module
 class HybridAdam(torch.optim.Optimizer):
+    """Implements Adam algorithm.
+
+    Support both GPU and CPU. Requires ColossalAI to be installed via
+    ``pip install .``
+
+    This version of Hybrid Adam is an hybrid of CPUAdam and FusedAdam.
+      * For updates on CPU, it uses CPUAdam.
```
Comment (feifeibear) [inline colossalai/nn/optimizer/hybrid_adam.py:16]: what tensors on CPU?

File: colossalai/nn/optimizer/cpu_adam.py (hunk 1 above)
Comment (Gy-Lu) [reply colossalai/nn/optimizer/cpu_adam.py:7]: is there any question?

File: colossalai/nn/optimizer/cpu_adam.py (hunk 2 above)
Comment (Gy-Lu) [reply colossalai/nn/optimizer/cpu_adam.py:11]: I will make it more detailed.

File: colossalai/nn/optimizer/cpu_adam.py (hunk 3 above)
Comment (Gy-Lu) [reply colossalai/nn/optimizer/cpu_adam.py:17]: Got it!

File: colossalai/nn/optimizer/cpu_adam.py (hunk 4 above)
Comment (Gy-Lu) [reply colossalai/nn/optimizer/cpu_adam.py]: I have updated the code and declared it.

**Review by feifeibear — APPROVED** [review]

**Pull request merged**